release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -202,6 +202,13 @@ For more information, see _Managing machines with the Cluster API_.
 [id="ocp-4-16-nodes_{context}"]
 === Nodes
 
+[id="ocp-4-16-must-gather-directory_{context}"]
+==== Information added to directory when using `oc adm must-gather`
+
+This release adds more information to the must-gather directory when the `oc adm must-gather` command is used. Now, the `oc` binary version and the logs generated by the `oc adm must-gather` are added to the must-gather directory. This helps identify issues that may stem from using a specific verions of `oc`, as well as listing what image was used, and if any data could not be gathered.
+
+For more information, see xref:../support/modules/about-must-gather.adoc#about-must-gather-tool[About the must-gather tool].
```

Comment:
I'm assuming you're on it already since the build is broken, but this xref isn't constructed properly. You don't list the "modules/" directory or the module file name.

I find the easiest way (for me at least) to figure out the xref is to start with the link in the preview or live docs, like: 

```
https://docs.openshift.com/container-platform/4.15/support/gathering-cluster-data.html#about-must-gather_gathering-cluster-data
```

Then get rid of the front part and put the appropriate number of "../" to back up, like:

```
../support/gathering-cluster-data.html#about-must-gather_gathering-cluster-data
```
Then change the .html to .adoc:
```
../support/gathering-cluster-data.adoc#about-must-gather_gathering-cluster-data
```
Then plop that into the xref syntax:
```
xref:../support/gathering-cluster-data.adoc#about-must-gather_gathering-cluster-data[About the must-gather tool]
```
Hope that helps!

More info in the guidelines is here: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#internal-cross-references

Reply:
I'm saving this in a doc for myself, it's gold!